profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/perimeters.tf

```diff
@@ -30,6 +30,25 @@ resource "google_folder_iam_binding" "admin-bucket-writer" {
   members = each.value.members
 }
 
+# BigQuery roles are necessary to support genomics extraction jobs, which bill
+# BigQuery costs to the researcher project. Owners will have these roles, so
+# this is well aligned with the above premise.
+resource "google_folder_iam_binding" "admin-bigquery-job-user" {
+  for_each = var.folder_admins
+
+  folder = google_folder.folder[each.key].name
+  role = "roles/bigquery.jobUser"
+  members = each.value.members
+}
+
+resource "google_folder_iam_binding" "admin-bigquery-read-session-user" {
+  for_each = var.folder_admins
+
+  folder = google_folder.folder[each.key].name
+  role = "roles/bigquery.readSessionUser"
```

Comment:
I've been granting `roles/bigquery.user` when I was doing my tests. We may be able to cut down on the granted permissions by combining some of the less permissive ones but I think we're still missing a few here. At the very least, I think we're missing `bigquery.datasets.create` and `bigquery.datasets.get`.

Reply:
This is granting permission on the researcher projects. We shouldn't need to get or create BigQuery datasets within those projects - we just need to be able to run queries / extractions that are billed under them.

I started with bigquery.user, but realized it allows creating datasets, which is excessive privilege here

Reply:
Ah, ok. I was thinking the temp tables would be created in the researcher project but I see they're supposed to stay within VPC-SC project.